src/Integration/Integration.csproj

```diff
@@ -59,19 +60,6 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="StrongNamer" Version="0.0.8" />
-  </ItemGroup>
-  <ItemGroup Label="Alm authentication">
-    <!-- The package from MS isn't signed so we need to reference the StrongName package
-         and conditionally set the DisableStrongName properties -->
-    <PackageReference Include="Microsoft.Alm.Authentication" Version="4.0.0.1" IncludeAssets="compile" />
```

Comment:
For some reason this reference was set the `IncludeAssets` property, whereas references in the test projects didn't.
However, it isn't required.